projects/cm_mcu/CommandLineTask.c

```diff
@@ -156,62 +156,88 @@ typedef struct __attribute__((packed)) {
 
 extern struct dev_i2c_addr_t pm_addrs_dcdc[];
 
+static BaseType_t reset_all(int argc, char **argv, char *m)
+{
+  static int which = 0;
+  static int page_d = 0;
+  for (; which < N_PM_ADDRS_DCDC; ++which) {
+    for (; page_d < 2; ++page_d) { // for reading two pages per device
+      bool reset = false;
+      reset = true;
+      uint8_t sn[32];
+      snapdump(&pm_addrs_dcdc[which], page_d, sn, reset);
+    }
+  }
+  return pdFALSE;
+}
+
 static BaseType_t snapshot(int argc, char **argv, char *m)
```

Comment:
I think you are not changing this function, right, since you are now introducing a new function? So can we just revert it to its original version?

Reply:
Yes, I did not add a new functionality to snapshot but instead made a new reset function. I thought that was what you suggested.